### DISCUSSION TITLE
suggestion

### DISCUSSION THREAD

**Discussion by Tiptup300** [TeamNewPipe/NewPipe#10608] [Questions]
I'll keep things brief,

With the timing of YouTube making that change causing a lot more users and the number of births since COVID. Now would be great to have an actual kid metering system. YouTube kids is garbage it only does 30min, it's bad on purpose.

If I could get YouTube videos a day, week, month for a specific profile that would do the trick.

**Comment by opusforlife2** [comment]
Are you using a translation software? Could you try a different one?
